modules/generate-verify/02_mod.mk

```diff
@@ -15,17 +15,19 @@
 .PHONY: generate
 ## Generate all generate targets.
 ## @category [shared] Generate/ Verify
-generate: $(shared_generate_targets)
+generate: $$(shared_generate_targets)
 
 verify_script := $(dir $(lastword $(MAKEFILE_LIST)))/util/verify.sh
 
 # Run the supplied make target argument in a temporary workspace and diff the results.
 verify-%: FORCE
 	$(verify_script) $(MAKE) -s $*
 
+verify_generated_targets = $(shared_generate_targets:%=verify-%)
```

Comment:
@ThatsMrTalbot what is the reason this is moved to a new variable?
Does that not cause the variable to be evaluated directly (not on second pass)?